.github/workflows/pkgci_regression_test_amdgpu_rocm.yml

```diff
@@ -75,3 +75,77 @@ jobs:
+      VENV_DIR: ${{ github.workspace }}/venv
+      IREE_TEST_FILES: ~/iree_tests_cache
+    steps:
+      - name: Checking out IREE repository
+        uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
+        with:
+          submodules: false
+      - uses: actions/setup-python@61a6322f88396a6271a6ee3565807d608ecaddd1  # v4.7.0
+        with:
+          # Must match the subset of versions built in pkgci_build_packages.
+          python-version: '3.11'
+      - uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a  # v3.0.2
+        with:
+          name: linux_x86_64_release_packages
+          path: ${{ env.PACKAGE_DOWNLOAD_DIR }}
+      - name: Setup venv
+        run: |
+          ./build_tools/pkgci/setup_venv.py ${VENV_DIR} \
+            --artifact-path=${PACKAGE_DOWNLOAD_DIR} \
+            --fetch-gh-workflow=${{ inputs.artifact_run_id }}
+
+      # Out of tree tests
+      - name: Check out external TestSuite repository
+        uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
+        with:
+          repository: nod-ai/SHARK-TestSuite
+          ref: 320edabdfba7f5cf46e5cd88569fca52308d8988
+          path: SHARK-TestSuite
+          submodules: false
+          lfs: true
+      - name: Install external TestSuite Python requirements
+        run: |
+          source ${VENV_DIR}/bin/activate
+          python -m pip install -r SHARK-TestSuite/iree_tests/requirements.txt
+      - name: Download remote files for real weight model tests
+        run: |
+          source ${VENV_DIR}/bin/activate
+          python SHARK-TestSuite/iree_tests/download_remote_files.py --root-dir pytorch/models
+      - name: Run external tests - Models with real weights
+        run: |
+          source ${VENV_DIR}/bin/activate
+          pytest SHARK-TestSuite/iree_tests/pytorch/models \
+            -rpfE \
+            -k real_weights \
+            --no-skip-tests-missing-files \
+            --capture=no \
+            --timeout=1200 \
+            --retries 2 \
+            --retry-delay 5 \
+            --durations=0 \
+            --config-files=build_tools/pkgci/external_test_suite/gpu_rocm_models.json
+      - name: "Running real weight model tests scheduled unet"
+        run: |
+          source ${VENV_DIR}/bin/activate
+          pytest SHARK-TestSuite/iree_tests/pytorch/models/sdxl-scheduled-unet-3-tank \
+            -rpfE \
+            -k real_weights \
+            --no-skip-tests-missing-files \
+            --capture=no \
+            --timeout=1200 \
+            --retries 2 \
+            --retry-delay 5 \
+            --durations=0 \
+            --config-files=build_tools/pkgci/external_test_suite/sdxl_scheduled_unet_gpu_rocm.json
+      - name: "Running SDXL rocm pipeline benchmark"
+        run: |
+          source ${VENV_DIR}/bin/activate
+          bash SHARK-TestSuite/iree_tests/benchmarks/benchmark_sdxl_rocm.sh
```

Comment:
Let's keep an eye on this and explore ways to improve the ergonomics.

Brainstorming:

* Document what specifically is being measured (what device, what program, what flags)
* Documentation can be in a comment here, included in logs, hosted on wiki / developer doc / readme file, etc.
* We could plug in to IREE's existing benchmarking infrastructure to get data into https://perf.iree.dev/ and PR comments, but setup there is tricky
* A lighter weight option than posting a PR comment is adding a job summary: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary. We use that for e.g.  https://github.com/iree-org/iree/actions/runs/8840713851?pr=17183#summary-24276534845
![image](https://github.com/iree-org/iree/assets/4010439/303adc5a-5fd3-4aa9-8b91-605dcede0ac7)

Reply:
Yeah sure, definitely improvements to be made here